devs/docs/basics.rst

```diff
@@ -66,6 +66,10 @@ Build a tarball like so::
 
     $ ./gradlew distTar
 
+Build a tarball of the free Community Edition like so::
```

Comment:
I am wondering if we should remove the word `free` to align with what we market as options here:
https://crate.io/products/cratedb-editions